server/src/migrations/20191010225527-create-hacker-profile.js

```diff
@@ -0,0 +1,53 @@
+export default {
+	up: (queryInterface, Sequelize) => {
+		return queryInterface.createTable("HackerProfile", {
+			id: {
+				allowNull: false,
+				autoIncrement: true,
+				primaryKey: true,
+				type: Sequelize.INTEGER
+			},
+			userId: {
+				type: Sequelize.INTEGER,
+				references: {
+					model: "Users",
+					key: "id"
+				},
+				allowNull: false
+			},
+			gender: {
+				type: Sequelize.ENUM,
+				values: ["male", "female", "other"]
+			},
+			ethnicity: {
+				type: Sequelize.STRING
+			},
+			major: {
+				type: Sequelize.STRING
+			},
+			minor: {
+				type: Sequelize.STRING
+			},
+			resume: {
+				type: Sequelize.STRING
+			},
+			skills: {
+				type: Sequelize.STRING
+			},
+			interests: {
+				type: Sequelize.STRING
+			},
```

Comment:
I'd like to have nullability maintain semantic meaning as best we can, it may be worth talking about which of these attributes we think are non-negotiable ( resume???)